bin/spryk-run

```diff
@@ -1,19 +1,67 @@
 #!/usr/bin/env php
 <?php
 
-include $_composer_autoload_path ?? getcwd() . '/vendor/autoload.php';
-
 use SprykerSdk\Kernel;
 use Symfony\Bundle\FrameworkBundle\Console\Application;
 
-define('APPLICATION_ROOT_DIR', getcwd() . DIRECTORY_SEPARATOR);
-define('APPLICATION_SOURCE_DIR', APPLICATION_ROOT_DIR . DIRECTORY_SEPARATOR . 'src' . DIRECTORY_SEPARATOR);
-define('APPLICATION_ENV', getenv('APPLICATION_ENV') ?? 'prod');
-define('APPLICATION', 'Spryk');
-define('APPLICATION_STORE', '');
+(function () {
```

Comment:
This is always the same?
Could we put this into a runtime.php etc
And just require_once() this in each file to keep things dry?